testing/zerotier-one/zerotier-one.initd

```diff
@@ -1,11 +1,14 @@
 #!/sbin/openrc-run
 
-depend() {
-    provide net
-    use network
-    before firewall netmount
-}
-
+name=zerotier-one
```

Comment:
Setting `$name` shouldn't be explicitly needed if it's equivalent to the filename.